squire_sdk/src/server/tournaments.rs

```diff
@@ -24,10 +25,31 @@ pub fn get_routes_and_init<S: ServerState>(state: S) -> Router<S> {
 
 pub fn get_routes<S: ServerState>() -> Router<S> {
     Router::new()
+        .route(
+            LIST_TOURNAMENTS_ENDPOINT.as_str(),
+            get(get_tournament_list::<S>),
+        )
         .route(GET_TOURNAMENT_ENDPOINT.as_str(), get(get_tournament::<S>))
         .route(SUBSCRIBE_ENDPOINT.as_str(), get(join_gathering::<S>))
 }
 
+pub async fn get_tournament_list<S>(
```

Comment:
Can you add a doc comment that includes the different ways this function can be called from API strings? Some examples would be nice. Ex: what does `api/v1/tournaments/list/` return?